### PR TITLE
refactor(directives): Drop ng- prefix from all angular directives and re...

### LINKED ISSUE #532: Drop ng- prefix from all angular directives and rename NgRepeat to Foreach
- [ ] `NgRepeat` => `Foreach`
- [ ] `NgSwitch` => `Switch`
- [ ] `NgIf` => `If`

### PR DESCRIPTION
**Pull request by marclaval**
...name NgRepeat to Foreach

fixes #532 

I had to keep `ng_` in the file names as using reserved keywords makes Dart build to fail.

### CODE REVIEW

File: modules/directives/directives.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export * from './src/ng_if';
```
Comment (mhevery) [inline modules/directives/directives.js]: rename `ng_*.dart` to `*.dart`

File: modules/directives/src/ng_foreach.js [hunk 2]
```diff
@@ -6,12 +6,12 @@ import {isPresent, isBlank} from 'facade/src/lang';
 import {ListWrapper} from 'facade/src/collection';
 
 @Template({
-  selector: '[ng-repeat]',
+  selector: '[foreach]',
```
Comment (mhevery) [inline modules/directives/src/ng_foreach.js]: the selector should be `[foreach][in]` since foreach without `in` is useless.

**Comment by mlaval** [comment]
Clean up done

File: modules/core/docs/01_templates.md [hunk 3]
```diff
@@ -345,20 +345,20 @@ Example of conditionally included template:
...
 Hello {{user}}!
-<div template="ng-if: isAdimnistrator">
+<div template="if: isAdimnistrator">
   ...administrator menu here...
 </div>
 ```
 
-In the above example the `ng-if` instantiator determins if the child view (an instance of the child template) should be
-inserted into ther root view. The `ng-if` makes this decision based on if the `isAdimnistrator` binding is true.
+In the above example the `if` instantiator determins if the child view (an instance of the child template) should be
```
Comment (gkalpak) [inline modules/core/docs/01_templates.md]: Typo: _determines_

**Pull request closed without merging**